### DISCUSSION TITLE
Problemas para acceder al formulario de entrega.

### DISCUSSION THREAD

**Discussion by AlejandroGarduno** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#267] [Q&A] [answered]
Hola, buenas noches.

He tratado de abrir el formulario de entrega pero me aparece este mensaje:

![image](https://user-images.githubusercontent.com/99100475/156909129-aa2faa5d-ee82-4259-990e-bd1dd63a01e1.png)

¿Alguien ha tenido problemas para ver el formulario de entrega?

Muchas gracias.

Saludos

**Comment by hackit92** [comment, marked as answer]
Asegurate de que la cuenta de microsoft con la que esta iniciada sesión sea la de launchx y no la de tu correo personal o empresarial, estoy seguro que de ahí viene el problema.

**Comment by RicardoCampos12** [comment]
Hola, Intenta abrirlo desde incognito para que te pida iniciar sesion y te asegures de que sea con tu cuenta de Innovaccion, a mi me pasa eso por que luego entra con el correo de la escuela pero asi se soluciona

**Comment by AlejandroGarduno** [comment]
Muchas gracias!
